spec/cypress/integration/accessibility.js

```diff
@@ -12,6 +12,9 @@ describe("Accessibility", () => {
     cy.get('[name="user[email]"]').type("doesntexist@example.com{enter}");
     cy.get("h1").should("contain", "You do not have an account");
     cy.checkA11y();
+    cy.appEval("ActionMailer::Base.deliveries").then((result) => {
```

Comment:
This is neat! Imo though, it should probably be separated out into a command to make it a bit more readable - e.g. `cy.appSentEmails().then(` or something